src/prefect/cli/agent.py

```diff
@@ -106,15 +107,19 @@ async def start(
 
     if not hide_welcome:
         if api:
-            app.console.print(f"Starting agent connected to {api}...")
+            app.console.print(
+                f"Starting v{prefect.__version__} agent connected to {api}..."
+            )
         else:
-            app.console.print("Starting agent with ephemeral API...")
+            app.console.print(
+                f"Starting v{prefect.__version__} agent with ephemeral API..."
+            )
 
     async with OrionAgent(work_queues=work_queues) as agent:
         if not hide_welcome:
             app.console.print(ascii_name)
             app.console.print(
-                "Agent started! Looking for work from "
+                f"Agent started! Looking for work from "
```

Comment:
You don't actually need this `f` here.

Reply:
🤦 this is evidence that I spent a lot of time trying to figure out where to put this information without printing a new line